spec/models/ems_refresh_spec.rb

```diff
@@ -279,5 +279,14 @@ def assert_queue_item(expected_targets)
       EmsRefresh.queue_merge([vm], ems)
       expect(MiqQueue.count).to eq(1)
     end
+
+    describe ".create_refresh_task" do
```

Comment:
right, naming of those 2 methods sounds like this test should be under `queue_refresh_task` context, but `.create_refresh_task` was not invoked from `.queue_refresh_taske`,  it called only once from  `.queue_merge` https://github.com/yrudman/manageiq/blob/af182ea3587b445879185490efee03fafc3744ec/app/models/ems_refresh.rb#L181.
This is the reasoin why it was put under `.queue_merge` context,
I thought it would be confusing to put test for method under context where it was not invoked.

Reply:
Hm we still call queue_merge from queue_refresh_task though so it should be fine there.

Reply:
ok, will move new test under `queue_refresh_task` context.